tests/test_CI.py

```diff
@@ -1,9 +1,10 @@
 import numpy as np
 import matplotlib.pyplot as plt
 
+
 def test_function():
     return np.sum(np.random.rand(5))
 
 
 if __name__ == "__main__":
-    test_function()
\ No newline at end of file
+    test_function()
```

Comment:
Do we need that now?

Reply:
No I don't think so. This file is not created by me at first and I just keep it.